lib/ramble/ramble/workspace/workspace.py

```diff
@@ -281,17 +281,17 @@ def active_workspace():
 
 def get_workspace_path():
     """Returns current directory of ramble-managed workspaces"""
-    path_in_config = ramble.config.get('config:workspace_dirs')
+    path_in_config = ramble.config.get('config:workspace_dirs', scope='user')
     if not path_in_config:
-        path_in_config = '$ramble/var/ramble/GET_WORKSPACE_ERROR/'
+        path_in_config = ramble.config.get('config:workspace_dirs', scope='_builtin')
 
     wspath = ramble.util.path.canonicalize_path(path_in_config)
     return wspath
 
 
 def set_workspace_path(dirname):
     """Sets the parent directory of ramble-managed workspaces"""
-    ramble.config.set('config:workspace_dirs:', dirname)
+    ramble.config.set('config:workspace_dirs', dirname, scope='user')
```

Comment:
Here I also don't love setting the scope explicitly. Maybe we don't need `set_workspace_path` anymore, and we should just use contextual config overrides where ever we have this.